### PR TITLE
Added exception handling when `GetConsoleMode` API fails

### PR DESCRIPTION
**Pull request by kuqin12**
## Description

This change fixes a pipeline break after moving to QEMU v9.0.0 by adding a try-except routine to handle the case on the server builds.

- [ ] Impacts functionality?
  - **Functionality** - Does the change ultimately impact how firmware functions?
  - Examples: Add a new library, publish a new PPI, update an algorithm, ...
- [ ] Impacts security?
  - **Security** - Does the change have a direct security impact on an application,
    flow, or firmware?
  - Examples: Crypto algorithm change, buffer overflow fix, parameter
    validation improvement, ...
- [ ] Breaking change?
  - **Breaking change** - Will anyone consuming this change experience a break
    in build or boot behavior?
  - Examples: Add a new library class, move a module to a different repo, call
    a function in a new library class in a pre-existing module, ...
- [ ] Includes tests?
  - **Tests** - Does the change include any explicit test code?
  - Examples: Unit tests, integration tests, robot tests, ...
- [ ] Includes documentation?
  - **Documentation** - Does the change contain explicit documentation additions
    outside direct code modifications (and comments)?
  - Examples: Update readme file, add feature readme file, link to documentation
    on an a separate Web page, ...

## How This Was Tested

This change was tested locally and on the pipeline build.

## Integration Instructions

N/A

### CODE REVIEW

**Review by apop5 — APPROVED** [review]

File: Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py [hunk 1]
```diff
@@ -231,7 +231,10 @@ def Runner(env):
         if os.name == 'nt' and qemu_version[0] >= '8':
             import win32console
             std_handle = win32console.GetStdHandle(win32console.STD_INPUT_HANDLE)
-            console_mode = std_handle.GetConsoleMode()
+            try:
+                console_mode = std_handle.GetConsoleMode()
+            except:
```
Comment (Javagedes) [inline Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py:236]: needs to be `except Exception`

https://docs.astral.sh/ruff/rules/bare-except/

File: Platforms/QemuSbsaPkg/Plugins/QemuRunner/QemuRunner.py [hunk 2]
```diff
@@ -147,7 +147,10 @@ def Runner(env):
         if os.name == 'nt' and qemu_version[0] >= '8':
             import win32console
             std_handle = win32console.GetStdHandle(win32console.STD_INPUT_HANDLE)
-            console_mode = std_handle.GetConsoleMode()
+            try:
+                console_mode = std_handle.GetConsoleMode()
+            except:
```
Comment (Javagedes) [inline Platforms/QemuSbsaPkg/Plugins/QemuRunner/QemuRunner.py:152]: same here - `except Exception`

**Review by Javagedes — CHANGES_REQUESTED** [review]

File: Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py (hunk 1 above)
Comment (kuqin12) [reply Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py]: Thanks. Updated.

File: Platforms/QemuSbsaPkg/Plugins/QemuRunner/QemuRunner.py (hunk 2 above)
Comment (kuqin12) [reply Platforms/QemuSbsaPkg/Plugins/QemuRunner/QemuRunner.py]: Updated.

**Review by Javagedes — APPROVED** [review]

**Pull request merged**